src/sentry/snuba/metrics/naming_layer/mri.py

```diff
@@ -83,3 +83,5 @@ class TransactionMRI(Enum):
     TOLERATED = "e:transactions/tolerated@none"
     APDEX = "e:transactions/apdex@ratio"
     MISERABLE_USER = "e:transactions/user.miserable@none"
+    ALL_USER = "e:transactions/user.all@none"
```

Comment:
nit: For sessions, we have `ERRORED_USER_ALL`. Should we call this one `USER_ALL`?

Reply:
I am following this https://github.com/getsentry/sentry/blob/5e9a29d87613d21c8e12403972b15bcb88538cc0/src/sentry/snuba/metrics/naming_layer/mri.py#L40

`ERRORED_USER_ALL` is just an internal name added for to simplify computation, and I chose this name for lack of better naming but I'd say if we want to be consistent we should probably follow the link I referenced

Reply:
OK makes sense.